modules/users/client/controllers/admin/view-applicants.client.controller.js

```diff
@@ -0,0 +1,45 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('users.admin')
+    .controller('ViewApplicantsController', ViewApplicantsController);
+
+  ViewApplicantsController.$inject = ['$scope', '$filter', '$window', 'ApplicantsService', 'Notification'];
+
+
+  function ViewApplicantsController($scope, $filter, $window, ApplicantsService, Notification) {
+    var vm = this;
+    ApplicantsService
+      .query(function (data) {
+        vm.unapprovedUsers = data;
+      });
+
+
+    vm.removeApplicant = function (user) {
+      if ($window.confirm('Are you sure you want to delete this user?')) {
+        if (user) {
+          vm.unapprovedUsers.splice(vm.unapprovedUsers.indexOf(user), 1);
+          ApplicantsService.remove(user);
+          Notification.success('User deleted successfully!');
+        }
+      }
+    };
+
+    vm.approvalSwap = function (user) {
+      if (user) {
+          var newUser = user;
```

Comment:
Your database call might not be working because newUser points to user. I don't really understand why we need to "swap" anything though. A better name for this function is approveApplicant

Reply:
Yeah, the naming convention is shit, I coded it to behave differently but decided I liked it better as a splice and not a toggle on the frontend.